src/coreclr/vm/comsynchronizable.h

```diff
@@ -52,13 +52,6 @@ friend class ThreadBaseObject;
         ThreadAbortRequested = 128,
     };
 
-    enum
```

Comment:
This enum was redundant. I have deleted it and the remapping code for enum values.